### PR TITLE
[8.0] fix for show installations

### PR DESCRIPTION
**Pull request by fstagni**
(not sure notes are needed)

### CODE REVIEW

File: src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def do_show(self, args):
                         for component in components:
                             record = []
                             if rDict[compType][system][component]["Installed"]:
-                                module = str(rDict[compType][system][component]["Module"])
+                                module = str(rDict[compType][system][component]["DIRACModule"])
```
Comment (chaen) [inline src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py:201]: does that work for extensions ?

Comment (fstagni) [reply src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py:201]: I don't see why not.

File: src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py [hunk 2]
```diff
@@ -189,7 +189,7 @@ def do_show(self, args):
             if not result["OK"]:
                 self._errMsg(result["Message"])
             else:
-                fields = ["System", "Name", "Module", "Type", "Setup", "Installed", "Runit", "Uptime", "PID"]
+                fields = ["DIRACSystem", "Name", "DIRACModule", "Type", "Setup", "Installed", "Runit", "Uptime", "PID"]
```
Comment (atsareg) [inline src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py]: This is only for display, no point to make it more complicated

Comment (fstagni) [reply src/DIRAC/FrameworkSystem/Client/SystemAdministratorClientCLI.py]: agree, corrected.

**Comment by fstagni** [comment]
Review OK now?

**Review by chrisburr — APPROVED** [review]

**Pull request merged**